src/components/ResumePage/element/Template1.js

```diff
@@ -37,30 +42,33 @@ const Template1 = () => {
       >
         <div ref={printElem}>
           {/* Main Header section which contain image & some personal information */}
-          <HStack justifyContent="space-between">
-            <VStack m={4} alignItems="flex-start" spacing={0.5}>
+          <HStack justifyContent="space-between"  >
+            <VStack m={4} alignItems="flex-start" spacing={0.5} style={HstackStyle}>
               <Heading as="h4" size="md">
                 {about.name ? about.name : "Harry Potter"}
               </Heading>
               <Text color={theme} fontWeight={"bold"}>
                 {about.role ? about.role : "Full Stack Web Developer"}
               </Text>
-              <HStack justifyContent={"space-between"}>
-                <HStack spacing={1}>
+              <HStack 
+                justifyContent={"space-between"}
+                fontSize={{ base: '11px', md: '9px', lg: '10px' , xl: "18px" }}
+              >
+                <HStack spacing={.5}>
```

Comment:
Instead of making the font size smaller, which handles the preview as well as doesn't mess up the download format